### ISSUE TITLE
[IMPROVEMENT] Change styling on answer's page so answers don't overflow/overlap 

### ISSUE THREAD

**Issue by nathenialalleyne** [prakhartiwari0/Arito#36]
## What improvement are you suggesting?
If the answer given is too long, they expand past the answer box and overlap

## Why should make this improvement?
Clogs up answer page and looks terrible

## How can we make these improvements?
Either add an overflow scroll for the entire box or just for the answer portion
Wrap text to the next line (This can change the box size might be the less ideal solution)
![e0c724f373b1d4a5a164ba2d2949f84a](https://user-images.githubusercontent.com/113154849/203635200-83bac0a1-2184-43a2-9fe4-26776de6a997.png)

**Comment by prakhartiwari0** [comment]
The **biggest answer that possibly can be their** is the product of 99,999 x 99,999, which is **9999800001**
Here is a brief explanation: https://github.com/prakhartiwari0/Arito/issues/35#issuecomment-1325875153

![image](https://user-images.githubusercontent.com/65062036/203682352-94caccce-cd22-490b-b086-fd69ca10ddce.png)

And as you can see, it isn't causing problems (at least on desktop)

**Comment by prakhartiwari0** [comment]
I just checked this on mobile, it is all good there also! 
If you have any more queries or suggestions, share them and we will discuss here 😄

**Comment by awnnshikaa** [comment]
The answers are still overlaping. And it would look better if the answers are centered.


![Screenshot from 2023-01-04 21-12-53](https://user-images.githubusercontent.com/106824384/210593382-3c13b447-3ec6-4983-88e1-65f05b33912d.png)

**Comment by prakhartiwari0** [comment]
Thanks for your suggestion, you are right in the UI part, but I would like to know what are your thoughts on the alignment of answers.
Doesn't it matter (at least in addition & subtraction)? 🤔

Is there anything else we could do to prevent the overlapping of the answers?

Also, 
The largest answer you can get in the VERY HARD level, in Addition, is **19999**. You can see below that the alignment is a bit disturbed which we can adjust by reducing the font size or maybe changing some padding, margin or alignment properties.
![image](https://user-images.githubusercontent.com/65062036/210703514-cfeeb5ce-0967-4e73-82f7-651225e25765.png)




The largest answer you can get in the VERY HARD level in Multiplication is **99990000** (The overall largest answer). As you can see the same problem is happening here, I guess we need to reduce the font size of question numbers and answers.
![image](https://user-images.githubusercontent.com/65062036/210703453-f5469df8-88bc-4271-a799-632c22d8dda8.png)


I would love to know what according to you is a good thing to do, does alignment matter or not, and should we make them centered (disturbing the alignment which is not good at present also), or should we change the font-size, padding & margin?

**Comment by awnnshikaa** [comment]
I think the size of question and answer should be even, so rather than changing the font size we must center the alignment. I tried to center the alignment but the overflowing still persists, for that it would be much better to make the box sizes same.

**Comment by prakhartiwari0** [comment]
Are you saying to make the boxes have fixed width and height? @awnnshikaa

**Comment by awnnshikaa** [comment]
yes @prakhartiwari0

**Comment by prakhartiwari0** [comment]
> yes @prakhartiwari0

But isn't that bad for responsiveness? We already have used the grid layout for the div that contains all the boxes in this CSS https://github.com/prakhartiwari0/Arito/blob/main/css/main.css#L428
and have given `min-width` and `max-width` on the boxes here https://github.com/prakhartiwari0/Arito/blob/main/css/main.css#L436

**Comment by awnnshikaa** [comment]
Yeah so keeping the responsiveness I have tried to make a changes.file:///home/awnnsheca/Pictures/Screensfile:///home/awnnsheca/Pictures/Screenshots/Screenshot%20from%202023-01-05%2013-16-26.png
hots/Screenshot%20from%202023-01-05%2013-15-32.png

Does this look better?

**Comment by awnnshikaa** [comment]
> > yes @prakhartiwari0
> 
> But isn't that bad for responsiveness? We already have used the grid layout for the div that contains all the boxes in this CSS https://github.com/prakhartiwari0/Arito/blob/main/css/main.css#L428 and have given `min-width` and `max-width` on the boxes here https://github.com/prakhartiwari0/Arito/blob/main/css/main.css#L436


keeping the respo

![Screenshot from 2023-01-05 13-15-32](https://user-images.githubusercontent.com/106824384/210728537-8d1105c7-64b8-4aa9-83f1-f0d394a4c400.png)
nsiveness i've made a few changes. Does this look better?

**Comment by awnnshikaa** [comment]
file:///home/awnnsheca/Pictures/Screenshots/Screenshot%20from%202023-01-05%2013-16-26.png

**Comment by awnnshikaa** [comment]
file:///home/awnnsheca/Pictures/Screenshots/Screenshot%20from%202023-01-05%2013-16-26.png

**Comment by awnnshikaa** [comment]
file:///home/awnnsheca/Pictures/Screenshots/Screenshot%20from%202023-01-05%2013-14-41.png

**Comment by awnnshikaa** [comment]
![Screenshot from 2023-01-05 13-14-41](https://user-images.githubusercontent.com/106824384/210728939-d4b3f863-26ed-4c67-abc4-8f97d85bc305.png)

![Screenshot from 2023-01-05 13-16-26](https://user-images.githubusercontent.com/106824384/210728986-697f982b-1a2b-4351-8326-9d984fca19c3.png)

**Comment by prakhartiwari0** [comment]
Can you please share a proper screen recording where you demonstrate your modified code and how the website looks in all the different screen sizes (using dev tools)?

**Comment by awnnshikaa** [comment]
/home/awnnsheca/Videos/Screencasts/Screencast from 05-01-23 04:01:45 PM IST.webm

**Comment by awnnshikaa** [comment]
![Uploading Screenshot from 2023-01-05 16-05-29.png…]()

aligned and justified to center as well as increased the minmax grid size to 300px

**Comment by awnnshikaa** [comment]
![Screenshot from 2023-01-05 16-05-29](https://user-images.githubusercontent.com/106824384/210761439-ca751124-3b09-472b-a7c5-826246d7f6be.png)

aligned and justified to center as well as increased the minmax grid size to 300px
